ludwig/distributed/deepspeed.py

```diff
@@ -182,6 +187,17 @@ def create_checkpoint_handle(
     ) -> Checkpoint:
         return DeepSpeedCheckpoint(self, dist_model, optimizer, scheduler)
 
+    @classmethod
+    def extract_model_for_serialization(cls, model: nn.Module) -> Union[nn.Module, Tuple[nn.Module, List[Dict]]]:
```

Comment:
Oops! That was definitely my mistake (in my defense, someone merged the PR before I had a chance to test it).